packages/mutation-testing-elements/src/typings/prism.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'prismjs/components/prism-core' {
+  // Core has same exports as root prismjs export
```

Comment:
Please also add this to a PR here: 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/prismjs/index.d.ts